contrib/demo/ingress

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+DEMO_DIR="$( dirname "${BASH_SOURCE[0]}" )"
+. ${DEMO_DIR}/demo-magic
+
+ROOT_DIR="$( cd ${DEMO_DIR}/../.. && pwd)"
+CLUSTERS_DIR=${CLUSTERS_DIR:-${DEMO_DIR}/clusters/kind}
+
+TYPE_SPEED=30
+#PROMPT_AFTER=1
+DEMO_PROMPT="ingress-demo $ "
+
+function pause() {
+  if [[ -n "${NO_WAIT}" ]]; then
+    sleep 5
+  else
+    if [[ -n "${1-}" ]]; then
+      sleep "$1"
+    else
+      wait
+    fi
+  fi
+}
+
+export KUBECONFIG=${KUBECONFIG:-${ROOT_DIR}/.kcp/data/admin.kubeconfig}
+if ! kubectl get namespaces &>/dev/null; then
+  echo "kcp server not started, run 'bin/kcp start'"
+  exit 1
+fi
+
+kubectl config use-context admin  &>/dev/null
+
+kubectl wait crd/clusters.cluster.example.dev --for condition=established &>/dev/null
+kubectl wait crd/negotiatedapiresources.apiresource.kcp.dev --for condition=established &>/dev/null
+kubectl wait crd/apiresourceimports.apiresource.kcp.dev --for condition=established &>/dev/null
```

Comment:
If you wanted to, you could do this with a single wait command and pass all the CRDs to it, e.g.

```bash
crds=(
  crd/clusters.cluster.example.dev
  crd/negotiatedapiresources.apiresource.kcp.dev
  crd/apiresourceimports.apiresource.kcp.dev
)
kubectl wait --for condition=established "${crds[@]}" &> /dev/null
```